docs/conf.py

```diff
@@ -56,6 +56,7 @@ def setup(app):
     "site",
     "README.md",
     "release-notes/README.md",
+    "deploy/README.md",
```

Comment:
ah I did this but then index file was referring to it and I though that can't be accidental 🤔  lgtm